### PR TITLE
Required reviews: allow Reach to approve Publicize&Social changes

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

The Reach team have started to work on the Social plugin; they should be able to approve changes to their plugin, as well as to the Publicize package which they'll work on a lot.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much to test for now, once merged the Reach folks should be able to merge PRs like #23768 without approvers' approval.

### CODE REVIEW

**Review by danielpost — DISMISSED** [review]
Thanks, this will be helpful!

File: .github/files/required-review.yaml [hunk 1]
```diff
@@ -55,6 +55,15 @@
    - jetpack-search
    - jetpack-approvers
 
+# The Reach team reviews changes to the Social plugin, and the Publicize package.
+- name: Social
+  paths:
+   - 'projects/plugins/social/**'
+   - 'projects/packages/publicize/**'
```
Comment (anomiex) [inline .github/files/required-review.yaml:62]: Needs an entry above like line 26. The team may also need to be added to the list starting at line 14.

**Review by anomiex — COMMENTED** [review]
> Not much to test for now

You can test it with a PR based on this one, like #23776. https://github.com/Automattic/jetpack/runs/5817203972?check_suite_focus=true#step:3:22 there shows one of the issues indicated inline.

**Review by anomiex — APPROVED** [review]
Looks good now: https://github.com/Automattic/jetpack/runs/5817757519?check_suite_focus=true#step:3:21 and https://github.com/Automattic/jetpack/runs/5817757519?check_suite_focus=true#step:3:27

**Pull request merged**